uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -512,7 +512,7 @@ public function upgrade_1020() {
         'option_group_id' => 'ta_settings',
         'name' => 'days_to_create_a_document_clone',
         'label' => is_callable('t') ? t('Renewed document creation date offset (days)') : 'Renewed document creation date offset (days)',
-        'value' => 0,
+        'value' => 90,
```

Comment:
We shouldn't modify an upgrader that has already been ran on sites.. Rather we should create a new one to effect any change we need.